great_expectations/expectations/core/expect_column_values_to_be_in_type_list.py

```diff
@@ -323,7 +323,7 @@ def get_validation_dependencies(
     ):
         # this calls TableExpectation.get_validation_dependencies to set baseline dependencies
         # for the aggregate version of the expectation
-        dependencies = super(ColumnMapExpectation, self).get_validation_dependencies(
+        dependencies = super().get_validation_dependencies(
```

Comment:
We need to keep this as `super(ColumnMapExpectation, self)`, which calls `TableExpectation.get_validation_dependencies` instead of `ColumnMapExpectation.get_validation_dependencies`. This is because the map version of this expectation is only supported for Pandas, so we want the aggregate version for the other backends.

Reply:
Thanks, @roblim -- this is very useful!  In next commit.